tutorials/source-modeling/plot_dipole_orientations.py

```diff
@@ -118,6 +118,7 @@
 _, time_max = stc.get_peak(hemi='lh')
 brain_fixed = stc.plot(surface='white', subjects_dir=subjects_dir,
                        initial_time=time_max, time_unit='s', size=(600, 400))
+mne.viz.set_3d_view(figure=fig, azimuth=180, distance=0.25)
```

Comment:
This change looks unrelated to me, can you check? :-)

Reply:
Sure! My mistake! Thanks